apps/chat/src/components/Common/ErrorMessage.tsx

```diff
@@ -1,18 +1,34 @@
-import { IconExclamationCircle } from '@tabler/icons-react';
+import { IconAlertTriangle, IconExclamationCircle } from '@tabler/icons-react';
+
+import classNames from 'classnames';
 
 export interface Props {
   error?: string;
+  type?: 'error' | 'warning';
```

Comment:
use enum instead of the sa,e string values in several places e.g. as `ToastType`